recipes/contextvars/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "contextvars" %}
+{% set version = "2.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "e9f9c5763d5a2afa6e420218e53954bd3829fc2d8acd806b98139d28c362cdf9" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python >=3,<3.7
+
+test:
+  imports:
+    - contextvars
+
+about:
+  home: http://github.com/MagicStack/contextvars
+  license: Apache
+  license_family: APACHE
+  license_file: 'LICENSE'
+  summary: PEP 567 Backport
+  description: "Implements a backport of Python 3.7 contextvar module (see PEP 567)"
+  doc_url: ''
```

Comment:
I would remove this rather than putting an empty string.

Reply:
Done